src/user/room.ts

```diff
@@ -428,23 +428,33 @@ export class Room extends Accessory implements Sayable {
     textOrContactOrFileOrUrl : string | Contact | FileBox | UrlLink,
     mention?                 : Contact | Contact[],
   ): Promise<void> {
+
+    const replyToList: Contact[] = [].concat(mention as any || [])
+
+    const mentionAliasPromiseList = replyToList.length > 0
+                                    ? await Promise.all(
+                                      replyToList
+                                      .map(c => this.alias(c) || c.name()))
+                                    : []
+
+    const mentionAliasList: string[] = mentionAliasPromiseList
+                                        .filter(a => a !== null) as string[]
+
     log.verbose('Room', 'say(%s, %s)',
                                   textOrContactOrFileOrUrl,
-                                  Array.isArray(mention)
-                                  ? mention.map(c => c.name()).join(', ')
-                                  : mention ? mention.name() : '',
-                )
-    let text: string
+                                  mentionAliasList.length > 1
```

Comment:
`mentionAliasList.length > 1` is not necessary at here.

Reply:
mentionAliasPromiseList may contain ```string | null```, mentionAliasList should only contain ```string```. Therefore, I think the line 440 is the one being filter. 
However, you are right. I should give a good name for the one contain ```string | null```